mrp_bom_location/models/mrp_bom.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MrpBom(models.Model):
+    _inherit = "mrp.bom"
+
+    location_id = fields.Many2one(
+        related='picking_type_id.default_location_dest_id',
+        store=True,
+    )
+
+
+class MrpBomLine(models.Model):
+    _inherit = "mrp.bom.line"
+
+    location_id = fields.Many2one(
```

Comment:
If these fields are related, shouldn't these be readonly here ?

Reply:
Yes..